section3/exercises/loops-arrays.js

```diff
@@ -18,32 +18,52 @@ for (var i = 0; i < animals.length; i++) {
 
 // YOU DO: Write code below that iterates through a list of animals and prints "The
 // <animal> is awesome!" for each animal.
+var pets = ['hamster', 'doggy', 'cat', 'lizard']
 
-
+for (var i = 0; i < pets.length; i++) {
+  console.log('The ' + pets[i] + ' is awesome!')
+}
 
 // YOU DO: Create an array of foods and then iterate over that array to log "Add
 // <food> to shopping list" for each food item.
+var foods = ['milk', 'eggs', 'bread', 'cheese']
 
-
+for (var i = 0; i < foods.length; i++) {
+  console.log ('Add ' + foods[i] + ' to shopping list')
```

Comment:
Remember to be consistent with using `;` at the end of each line, since you used them with most of your syntax.